utils/general.go

```diff
@@ -134,6 +134,15 @@ func Int64InSlice(i int64, s []int64) bool {
 	return false
 }
 
+func StringInSlice(str string, s []string) bool {
+	for i := range s {
```

Comment:
Consider change the function body to:
return utils.Contains(utils.Is(s), str)

Or replace all StringInSlice references to use utils.Contains like we already do in other parts of the project when we check that some string exist is slice.